src/main/java/org/folio/des/scheduling/BursarExportDeleteScheduler.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.des.scheduling;
+
+import org.folio.des.domain.dto.ExportConfig;
+import org.folio.des.exceptions.SchedulingException;
+import org.folio.des.scheduling.quartz.QuartzExportJobScheduler;
+import org.folio.des.scheduling.quartz.converter.bursar.ExportConfigToBursarDeleteJobDetailConverter;
+import org.folio.des.scheduling.quartz.converter.bursar.ExportConfigToBursarDeleteTriggerConverter;
+import org.folio.des.scheduling.quartz.job.bursar.BursarDeleteJobKeyResolver;
+import org.quartz.Scheduler;
+import org.springframework.stereotype.Service;
+
+import lombok.extern.log4j.Log4j2;
+
+@Service
+@Log4j2
+public class BursarExportDeleteScheduler {
+  private final QuartzExportJobScheduler quartzExportJobScheduler;
+
+  public BursarExportDeleteScheduler(ExportConfigToBursarDeleteTriggerConverter exportConfigToBursarDeleteTriggerConverter, BursarDeleteJobKeyResolver bursarDeleteJobKeyResolver, ExportConfigToBursarDeleteJobDetailConverter exportConfigToBursarDeleteJobDetailConverter, Scheduler scheduler) {
+
+    this.quartzExportJobScheduler = new QuartzExportJobScheduler(scheduler,
+      exportConfigToBursarDeleteTriggerConverter, exportConfigToBursarDeleteJobDetailConverter, bursarDeleteJobKeyResolver);
+  }
+
+  public void scheduleBursarDeleteJob(ExportConfig exportConfig) {
+    try {
+      quartzExportJobScheduler.scheduleExportJob(exportConfig);
```

Comment:
Don't think it's a good idea to use QuartzExportJobScheduler here. There's nothing done with export configs (with its ids, schedules, scheduling/unscheduling handling), just old jobs are deleted once per day. Also additional converter classes had to be added in order to be able to use QuartzExportJobScheduler, which adds complexity and at the same time it's actually not needed here.  I'd prefer to keep it simple and use a plain quartz scheduler here. 

Also probably BursarExportDeleteScheduler isn't a right naming, aren't other job types deleted as well?